packages/core/src/bridge.ts

```diff
@@ -94,7 +94,9 @@ export const useWorldBridgeStore = create<WorldBridgeStore>((set, get) => ({
 			verificationState: VerificationState.WaitingForConnection,
 			connectorURI: `https://worldcoin.org/verify?t=wld&i=${request_id}&k=${encodeURIComponent(
 				await exportKey(key)
-			)}${bridge_url && bridge_url !== DEFAULT_BRIDGE_URL ? `&b=${encodeURIComponent(bridge_url)}` : ''}`,
+			)}${
+				bridge_url && bridge_url !== DEFAULT_BRIDGE_URL ? `&b=${encodeURIComponent(bridge_url)}` : ''
+			}&return_to=${encodeURIComponent(window.location.href)}`,
```

Comment:
should only be appended when tapping the universal link button, this will include it in the QR code as well